src/main/features/response/views/when-will-you-pay.njk

```diff
@@ -14,8 +14,13 @@
             novalidate>
         {{ csrfProtection(csrf) }}
 
+        <div class="notice">
+          <i class="icon icon-important"><span class="visually-hidden">{{ t('Warning') }}</span></i>
+          <strong class="bold-small">{{ t('You may receive a County Court Judgment (CCJ) if you don`t pay immediately.') }}</strong>
+        </div>
+
         {% if statementOfMeansIsApplicable %}
-          {% set hintText = t('You will need to give details of your finances to {{ claimantFullName }} to explain why.', { claimantFullName: claim.claimData.claimant.name }) %}
+          {% set hintText = t('You will need to give details of your finances to {{ claimantFullName }} to explain why you can`t pay immediately.', { claimantFullName: claim.claimData.claimant.name }) %}
```

Comment:
this is translated in the macro. Please remove the t()

Reply:
Actually it makes sense to do it here, otherwise we would not be able to pass claimant name. This way interpolated and translated string will be passed for another translation that will result in the same string returned.

Reply:
Ok